aimmo-game/service.py

```diff
@@ -46,7 +44,7 @@ def player_dict(avatar):
         'rotation': 0,
         "colours": {
             "bodyStroke": "#0ff",
-            "bodyFill": colour,
+            "bodyFill": "#%06x" % (avatar.player_id * 4999),
```

Comment:
a bit dangerous or is it just me?

Reply:
How is it different to the one we had before? I did open an issue about this though #369

Reply:
it's not different - was dangerous too

Reply:
I wouldn't worry too much about this if we are going to start using the 3d models, the colour is not passed to the client anyways. Actually that means that this can probably be removed

Reply:
Should I make an issue @mrniket?

Reply:
let's add something onto #369